modules/acquia_cms_headless/tests/src/Functional/DashboardApiDocumentTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+use Drupal\Tests\acquia_cms_headless\Traits\DashboardTableTrait;
+
+/**
+ * Tests headless dashboard API Document.
+ *
+ * @group acquia_cms
+ * @group acquia_cms_headless
+ * @group medium_risk
+ * @group push
+ */
+class DashboardApiDocumentTest extends DashboardWebDriverTestBase {
+
+  use DashboardTableTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected string $sectionTitle = "API Documentation";
+
+  /**
+   * {@inheritdoc}
+   */
+  protected string $sectionSelector = "#acquia-cms-headless-api-docs";
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getHeaders(): array {
+    return [];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function testButtons(): void {
+    $this->assertButton("Explore with Redoc");
+    $this->assertButton("Explore with Swagger UI");
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function testSection(): void {
+    $assertSession = $this->assertSession();
+
+    // Test API Documentation section exists, get API Documentation section.
+    $element = $assertSession->elementExists('css', $this->sectionSelector);
+
+    // Test OpenAPI Resources label exist.
+    $openApiResource = $element->find("css", '.headless-dashboard-openapi-resources > p > a');
+    $this->assertEquals('OpenAPI Resources', $openApiResource->getText());
+
+    // Test OpenAPI Resources link.
+    $buttonAction = $openApiResource->getAttribute('href');
```

Comment:
@deepakmishra2 Also test if link has attribute target and value as _blank.
<img width="1018" alt="Screenshot 2023-01-24 at 10 32 48 PM" src="https://user-images.githubusercontent.com/81069167/214358945-ce52b677-cfae-4fb1-985e-7cdb4cafc359.png">

Reply:
+1, for testing OpenAPI Resource link, create separate test function if possible.

Reply:
done